Documentation/bootstrapping-guidelines.md

```diff
@@ -0,0 +1,116 @@
+
+## Building a New .NET Major Version
+
+Building a new .NET major version is two stage process:
+
+**Stage 1:** Build bootstrapping .NET SDK for the targeted platform.
+
+1. Download a platform-native portable Microsoft-built version of the dotnet SDK for bootstrapping as well as the previously-source-built package archive.
+
+    ``` bash
+    ./prep.sh
+    ```
+
+1. Build the source built .NET SDK.
+
+    ``` bash
+    ./build.sh
+    ```
+
+**Stage 2:** Use the source built .NET SDK created in stage 1 to build .NET SDK from source. There is no need to run `prep.sh` in this stage.
+
+1. Extract your freshly-built stage 1 SDK to a convenient location.
+
+    ``` bash
+    tar -ozxf /<stage1-path>/artifacts/<arch>/Release/dotnet-sdk-<version>-<rid>-tar.gz -C <extracted-stage1-sdk-path>
+    ```
+
+1. Build the source built .NET SDK.
+
+    ``` bash
+    ./build.sh --with-sdk <extracted-stage1-sdk-path> --with-packages /<stage1-path>/obj/bin/<arch>/blob-feed/packages
+    ```
+
+## Building a Servicing Release of .NET
+
+Building a subsequent or servicing version of .NET requires that you have source built the previous version of .NET available as descibed in [Building New .NET Major Version](#building-a-new-net-major-version). If you already have the previous verions of the .NET SDK available, all you have to do is run the following build command.
+
+``` bash
+./build.sh --with-sdk <extracted-previously-source-built-sdk-path> --with-packages <extracted-previously-source-built-packages-path>
+```
+
+## Building for a New OS/Arch (Supported RID)
+
+**Supported OS**  
+[Check](https://learn.microsoft.com/en-us/dotnet/core/install/linux) officially supported linux distributions.  
+[List](https://github.com/dotnet/installer/blob/release/7.0.1xx/src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml#L12-L16) of Linux distribution officially building and testing using source-build.  
+Community building .NET for the [following](https://github.com/dotnet/source-build#net-in-linux-distributions) linux distributions.  If the RID for your platform is already in the [runtime RID graph](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json), you can source-build the SDK without any source changes.
+
+**Supported Archs**  
+Officially building and testing for the following architectures - x64, ARM64  
+Community building for following architectures - s390x, ARM32, ppc64le
+
+**Supported RIDs**
+You can see the list of supported RIDs and the RID graph in the [runtime.json](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json). Learn more about RID catalog in [here](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#linux-rids).
+
+**Refer section "Building New .NET Major Version" for the steps to build.**
+
+## Building for New OS (Unsupported)  
```

Comment:
I think the working in these headers needs to be clarified in regards to the use of unsupported and supported.

Reply:
Tried to clear this up with what we actually mean.